reference/forms/types/options/choice_translation_parameters.rst.inc

```diff
@@ -0,0 +1,52 @@
+choice_translation_parameters
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**type**: ``array``, ``callable``, ``string`` or :class:`Symfony\\Component\\PropertyAccess\\PropertyPath` **default**: ``[]``
+
+The choice values are translated before displaying it, so it can contain
+:ref:`translation placeholders <component-translation-placeholders>`.
+This option defines the values used to replace those placeholders. This can be
+an associative array where the keys match the choice keys and the values
+are the attributes for each choice, a callable or a property path
+(just like `choice_label`_).
+
+Given this translation message:
+
+.. code-block:: yaml
```

Comment:
What about adding the other translation formats as well?

Reply:
Done